plugins/pluginManager.js

```diff
@@ -828,13 +829,17 @@ var pluginManager = function pluginManager() {
                 return callback(errors);
             }
             var cwd = eplugin ? eplugin.rfs : path.join(__dirname, plugin);
-            exec('sudo npm install --unsafe-perm', {cwd: cwd}, function(error2) {
-                if (error2) {
-                    errors = true;
-                    console.log('error: %j', error2);
+            this.syncPlugins(this.getConfig("plugins"), function() {
```

Comment:
I meant we need to modify that specified code to enclose it in loadConfig call, not to bring it here. Here it won't do any good :)

So we need to revert the changes here and modify the code here: https://github.com/Countly/countly-server/blob/master/plugins/plugins/api/api.js#L44

to wrap syncPlugins around in loadConfigs callback

And by revert I mean only last change, not all changes :)